test/test-unit/Makefile

```diff
@@ -0,0 +1,16 @@
+BIN_DIR=../../build/test/test-unit
+TEST_BINARIES=read_bytes.test
+
+all: $(TEST_BINARIES)
+
+read_bytes.test: read_bytes.c unit.c
+	gcc -o $@ $^
```

Comment:
I guess `$(CC)` would be useful for ones, who have clang installed (but not gcc).

Reply:
Fixed.